tests/bench/re2s/README

```diff
@@ -0,0 +1 @@
+Adapted from https://github.com/twitter/reasonable-scala/, which is licensed under Apache v2.
```

Comment:
Maybe we should not include source code here and reference it via submodule instead.

/cc: @nicolasstucki @anatoliykmetyuk